.editorconfig

```diff
@@ -15,4 +15,22 @@ indent_size = 2
 indent_style = space
 
 [*.md]
-trim_trailing_whitespace = false
+max_line_length = 999
+tab_width = 2
+ij_continuation_indent_size = 2
```

Comment:
it looks like specific annotations for intellij ? so not sure if it should be there. It should contains only generic properties.

the formatter used is prettier so if it requires some formatting check/format it should probably goes there

Reply:
It's a way to achieve consistency. Some features work only on IntelliJ. I am happy to provide similar features for VSCode and other.

Reply:
Prettier is something I would like to explore more, but I am not sure it integrates as well with the IDE.

Reply:
NB: the Prettier plugin is not available on IntelliJ IDEA. https://youtrack.jetbrains.com/issue/WEB-59644/Provide-Prettier-plugin-on-IntelliJ-IDEA That's such bad news.

Reply:
I had a look at Prettier documentation: I have no doubts it is a splendid tool for the code (it enables me to dare to do some contributions), but we can't rely on it to lint/format Markdown in documentation. 

* It supports EditorConfig for basic stuff https://prettier.io/docs/en/configuration.html#editorconfig

* The only prose related option is better kept as default (preserve): https://prettier.io/docs/en/options.html#prose-wrap

We have yet to find a good way to reformat Markdown files without breaking nested blocks.